R/request.R

```diff
@@ -99,7 +99,14 @@ print.request <- function(x, ...) {
 
 request_prepare <- function(req) {
   req <- request_combine(request_default(), req)
-  req$options$customrequest <- req$method
+  switch(req$method,
+         "POST" = {
```

Comment:
I'm not sure what you mean by silent, but I don't think choice of parens matters.